instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/FieldMapper.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2;
+
+import io.opentelemetry.api.trace.Span;
+import java.util.function.Function;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import software.amazon.awssdk.core.SdkRequest;
+import software.amazon.awssdk.core.SdkResponse;
+
+class FieldMapper {
+
+  private final Serializer serializer;
+  private final MethodHandleFactory methodHandleFactory;
+
+  FieldMapper() {
+    serializer = new Serializer();
+    methodHandleFactory = new MethodHandleFactory();
+  }
+
+  FieldMapper(Serializer serializer, MethodHandleFactory methodHandleFactory) {
+    this.methodHandleFactory = methodHandleFactory;
+    this.serializer = serializer;
+  }
+
+  void mapFields(SdkRequest sdkRequest, AwsSdkRequest request, Span span) {
```

Comment:
Since this is a `FieldMapper`, `mapToAttributes` for the method name? I've read the word a few times now and every time I have to remember what it's mapping to

Reply:
Good idea!